files/slc_clean

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+show_help() {
+    echo 'Options:
+    -s, --svc               Service ID (default: 1)
+    -c, --count             Version kept (default: 2)
+    -h, --help              Show this help
+'
+    echo 'List Services:'
+    if hash slc 2>/dev/null; then
+        slc ctl ls
+    fi
+}
```

Comment:
Add some details - such as what to do with the list being printed out.

Include an example of command

Include a `-y` | `--yes` option to auto confirm delete (see below) - default to "no"